Makefile.am

```diff
@@ -65,6 +66,7 @@ EXTRA_DIST = \
 CLEANFILES = \
 	vdsm.spec \
 	$(DIST_ARCHIVES) \
+	$(wildcard $(RPM_TOPDIR)/*/*/*.rpm) \
```

Comment:
Does it works to remove the entire build directory or there is CLEANDIRS for this?